bibtex-actions.el

```diff
@@ -73,25 +126,55 @@
                (lambda (string predicate action)
                  (if (eq action 'metadata)
                      '(metadata
-                       ;; TODO (affixation-function . bibtex-actions--affixation)
+                       (where (,bibtex-actions-rich-ui)
```

Comment:
This is the other problematic line, @mohkale; probably the same problem.

I want this affixation function to run by default, but to allow a user to turn it off if they prefer.